techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -60,9 +60,8 @@ bundle agent doInventory
   methods:
     ignore_disable_inventory|!disable_inventory::
       "any" usebundle => doInventory_always;
-  reports:
     !(ignore_disable_inventory|!disable_inventory)::
-      "INFO: The file ${g.rudder_disable_inventory_file} is present. Rudder will not send an inventory. Use rudder agent inventory --force to override this temporarily";
+      "any" usebundle => rudder_common_report("Inventory", "log_info", "&TRACKINGKEY&", "inventory", "None", "The file ${g.rudder_disable_inventory_file} is present, so Rudder will not send an inventory. Use 'rudder agent inventory -f' to override this temporarily");
```

Comment:
Hang on, though. I don't think we want this to be a Rudder report. The goal here is that when you run the "rudder agent inventory" command, you get a human readable message telling you why the inventory wasn't made. Will this work OK with the Rudder report style?